### PR TITLE
Remove deprecated charts.d plugins served by native plugins.

### PR DESCRIPTION
**Pull request by Ferroin**
This removes three of the charts.d plugins that cover data collection that is now done in the netdata core, or the apps.plugin code.  From a practical perspective, there is no reason that anybody who is running a current version of netdata actually needs these given that the native versions in the netdata core and apps.plugin cover 100% of their functionality (and more).

Inspired by #4036.  I figured that since these three are quite clearly superseded by C code doing the same thing and have been for quite a while now, it should be reasonably uncontroversial to remove them.

### CODE REVIEW

**Comment by paulfantom** [comment]
LGTM

File: conf.d/charts.d.conf [hunk 1]
```diff
@@ -56,8 +56,3 @@
 # sensors=force
 # squid=force
 # tomcat=force
-
-# OLD MODULES THAT ARE NOW SERVED BY NETDATA DAEMON
-# cpu_apps=force
-# mem_apps=force
-# load_average=force
```
Comment (ktsaou) [inline conf.d/charts.d.conf]: Please don't remove these lines.
The next update of netdata will enable them, since they are already installed.

**Comment by paulfantom** [comment]
Can we add sth to installer/updater script to remove those files?

**Comment by Ferroin** [comment]
I'm with @paulfantom on this, I'd rather update the installer/updater to remove these files than keep the lines to disable them.  Especially if we're pushing for proper packaging, because package managers _will_ remove these files when you install a version of Netdata that does not include them.

**Comment by ktsaou** [comment]
hm... this could also be done for other obsolete files, like the `web/old` ones...
We could have `/installer/removed_obsolete_files.in` with autoconf formatting, a list of files to remove if they are present. This would need to be done in a way that we will accidentally remove any other files from the system....

**Comment by gmosx** [comment]
@Ferroin Seems like minor work is needed to unblock this issue, I am wondering if you could have a look.

**Comment by Ferroin** [comment]
I think it's largely just a case of the conversation dying out.  I generally agree that the installer needs to handle this long-term (especially with stuff like #4562 on the way), but I'm not sure about the best way to go about that.

Short term, just removing the changes to the config and adding a comment that these plugins have been removed upstream is probably sufficient for the PR to get merged.

This really does bring up one of the big reasons to have proper packaging, the package manager handles this crap for you.

**Comment by Ferroin** [comment]
Dead simple idea for handling this properly in the installer:

Uninstall everything except the updater script itself during the update right before we install everything.  We should actually be doing this anyway probably so that we're certain that we have the most up-to-date files (and we can afford to do it easily now that we don't keep default config in `/etc`).

**Comment by cakrit** [comment]
Created #6151 for @paulkatsoulakis to pick this up.

**Comment by paulkatsoulakis** [comment]
Closing this - i have migrated the branch as is to my own fork.
@Ferroin feel free to follow up on #6151 further about this, thanks for all the initial work on this one!

**Pull request closed without merging**